ruby/vampires.rb

```diff
@@ -46,7 +46,7 @@
 
 if insurance == "y"
   insurance_yes = insurance
-  puts "You would like to enroll."
+  puts "Thank you for enrolling."
 elsif insurance == "n"
   insurance_no = insurance
   puts "You do not want to enroll."
```

Comment:
It is always a good idea to try and let our loops break naturally.  Using `break` is an abrupt stop.

A good refactor suggestion for your loops would be to use a double conditional to break a while loop.

If you the user adds done or the allergy you are looking for you can end your loop.